DigitalLearningSolutions.Web/Controllers/Signposting/LinkLearningHubSsoController.cs

```diff
@@ -0,0 +1,58 @@
+﻿namespace DigitalLearningSolutions.Web.Controllers.Signposting
+{
+    using System;
+    using DigitalLearningSolutions.Data.Exceptions;
+    using DigitalLearningSolutions.Data.Models.Signposting.LinkLearningHubSso;
+    using DigitalLearningSolutions.Data.Services;
+    using DigitalLearningSolutions.Web.Helpers;
+    using DigitalLearningSolutions.Web.ViewModels.Signposting.LinkLearningHubSso;
+    using Microsoft.AspNetCore.Authorization;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.FeatureManagement.Mvc;
+
+    [FeatureGate(FeatureFlags.UseSignposting)]
+    [Route("Signposting/[Controller]")]

```

Comment:
Would there be any issue with making this route `Signposting/LinkLearningHubSso`?

Reply:
No issue. Done.